lstmcpipe/config/__init__.py

```diff
@@ -0,0 +1,10 @@
+import os
+from shutil import which
+
+
+def export_env(outdir='.'):
```

Comment:
I also investigated yesterday how to best do this. There is `pkg_resources.working_space` which includes all packages with name, version, path that are accessible to the current interpreter.:

```
from pkg_resources.working_set


for package in sorted(working_set, key=lambda p: p.project_name):
     print(package.project_name, package.version, package.module_path)
```

Reply:
awesome, thank you!